### PR TITLE
[bitnami/tomcat] add command option to image

### PR DESCRIPTION
**Pull request by loomsen**
**Description of the change**

Parameterize `image.command`, so we can run custom images with custom commands.
This will allow to install this chart with custom switches for the tomcat server.

For example
```
helm install tomcat bitnami/tomcat \
  --set image.command.0='java' \
  --set image.command.1='-jar' \
  --set image.command.2='-Dspring.profiles.active=development' \
  --set image.command.3='-Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true' \
  --set image.command.4='/app.jar'
```

**Benefits**

We will be able to pass custom commands to the tomcat image.

**Possible drawbacks**

-

**Applicable issues**

-

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi,

Thank you so much for the PR! For consistency with the rest of the charts, could you change `image.command` to `command` and set the YAML like this? 

```
          {{- if .Values.command }}
          command: {{- include "tomcat.tplValue" ( dict "value" .Values.command "context" $) | nindent 12 }}
          {{- end }}
```

**Comment by loomsen** [comment]
Hi @javsalgar,

If I understood correctly, the last PR should be what you are asking for?

Thank you for reviewing.

File: bitnami/tomcat/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: tomcat
-version: 6.3.17
+version: 6.3.18
```
Comment (javsalgar) [inline bitnami/tomcat/Chart.yaml:3]: Please bump the minor version as it is adding a feature

**Review by javsalgar — APPROVED** [review]
Thanks! LGTM. Minor comment

File: bitnami/tomcat/Chart.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/tomcat/Chart.yaml:3]: Never mind, I'll do it myself

**Pull request merged**